framework/src/modules/token/cc_commands/cc_transfer.ts

```diff
@@ -56,99 +53,109 @@ export class CCTransferCommand extends BaseCCCommand {
 
 	public init(args: { minBalances: MinBalance[]; supportedTokenIDs: Buffer[] }): void {
 		this._supportedTokenIDs = args.supportedTokenIDs;
-		this._minBalances = args.minBalances;
+		// this._minBalances = args.minBalances;
 	}
 
-	public async execute(ctx: CCCommandExecuteContext): Promise<void> {
-		const { ccm } = ctx;
+	public async verify(ctx: CCCommandExecuteContext): Promise<void> {
+		const { ccm, ccmSize } = ctx;
 		const methodContext = ctx.getMethodContext();
 		const { id: ownChainID } = await this._interopMethod.getOwnChainAccount(methodContext);
```

Comment:
ownChainID can be obtained from `init` function passed from module [here](https://github.com/LiskHQ/lisk-sdk/blob/feature/7211-update_token_module/framework/src/modules/token/module.ts#L159)